demos/speech-to-text/src/index.tsx

```diff
@@ -5,11 +5,13 @@ import App from './App';
 import { SpeechProvider } from "@speechly/react-client";
 import * as serviceWorker from './serviceWorker';
 import { LogKit } from "@speechly/logkit"
+import { DemoNavigation } from '@speechly/demo-navigation';
 
 ReactDOM.render(
   <React.StrictMode>
     <SpeechProvider appId="6f1c7eaa-53fa-495e-9319-4ceacfa88cfe">
       <LogKit appName="speech-to-text" appVersion={100} >
+        <DemoNavigation />
```

Comment:
This demo is embedded here: https://www.speechly.com/technology/voice-api/speech-to-text/

We probably don't want the navigation bar to appear there?